src/freertos_drivers/st/Stm32Can.cxx

```diff
@@ -435,31 +448,35 @@ void Stm32Can::sce_interrupt_handler()
         /* error interrupt has occured */
         CAN->MSR |= CAN_MSR_ERRI; // clear flag
 
+        bool cancel_queue = false;
+        
         if (CAN->ESR & CAN_ESR_EWGF)
         {
             /* error warning condition */
             state_ = CAN_STATE_BUS_WARNING;
-            CAN->ESR &= ~CAN_ESR_EWGF;
```

Comment:
Are we sure we want to remove this flag clearing, and the two below?

Reply:
I'll add this from the git commit to the PR description:
> Removes flag clear statements for error flags in the ESR register, because these flags are not writeable (can not be cleared in software).